controller/codebase.go

```diff
@@ -136,8 +136,8 @@ func (c *CodebaseController) Create(ctx *app.CreateCodebaseContext) error {
 	}
 	cheClient := che.NewStarterClient(c.config.GetCheStarterURL(), c.config.GetOpenshiftTenantMasterURL(), getNamespace(ctx))
 
-	stackID := cb.StackID
-	if cb.StackID == "" {
+	stackID := *cb.StackID
+	if *cb.StackID == "" {
```

Comment:
you should do this instead:
````
if cb.StackID == nil {
...
}
````
otherwise you'll get a `panic` if the `cb.StackID` is nil